_includes/_sass/includes/_docs-section.scss

```diff
@@ -18,11 +18,24 @@
 	.anchor-heading {
 		padding-top: 60px;
 		margin-top: -60px;
+
+		&::after {
```

Comment:
Sorry, can you make this a clickable element in the DOM? We want to be able to click the icon so the user can get the sharable URL